### ISSUE TITLE
[Adaptive Cards- Import- Expense Report]: Luminosity ratio of 'Reject' text is '2.0:1' which is less than 4.5:1 on Reject button under Expense Report Pane.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8475]
### Target Platforms

Android

### SDK Version

 2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

This bug is also found in FC-Jan22. Linking WI
Test Environment:
OS: Android 12
Device: S21+5g
Screen reader: Talkback
AdaptiveCard Android App version:  2023.03.31.1
--------

​Test Environment:​
Device : Android
Android Version : 10
Application: Adaptive Card
Application Version: 2021.06.04.1
Tool: Accessibility Insights for Android

Repro Steps:

1. Launch the Adaptive cards app on Android.
2.  Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Expense Report'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate all the controls under 'Visual'.
8. Navigate to 'Reject' button and observe the issue with the tool.

Actual Result:
Luminosity ratio of 'Reject' text is '2.0:1' which is less than 4.5:1 on Reject button under Expense Report Pane.

Expected Result:
Luminosity ratio of 'Reject' text should be equal or greater than 4.5:1 on Reject button under Expense Report Pane.

User Experience:
User with low vision might face difficulty to understand/visualize the button, If the color contrast ratio is less than 4.5:1.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![Android CCA Checker- Reject button](https://user-images.githubusercontent.com/89897257/232741973-7b1428f7-9b43-40c0-9d61-7f9350b33216.jpg)
![CCA of Reject text is with respect to background is low](https://user-images.githubusercontent.com/89897257/232741992-21c0319a-bcc4-478f-80dc-e51897725e3d.jpg)

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
duplicate issue of hostconfig configuration options.
